test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectExplorer/ObjectExplorerServiceTests.cs

```diff
@@ -130,19 +130,32 @@ FOR CREATE_DATABASE
             await RunTest(databaseName, query, "TestDb", async (testDbName, session) =>
             {
                 var tablesNode = await FindNodeByLabel(session.Root.ToNodeInfo(), session, SR.SchemaHierarchy_Tables);
-                var tableChildren = await _service.ExpandNode(session, tablesNode.NodePath);
+                var tableChildren = (await _service.ExpandNode(session, tablesNode.NodePath)).Nodes;
                 string dropTableScript = "Drop Table t1";
                 Assert.True(tableChildren.Any(t => t.Label == "dbo.t1"));
                 await TestServiceProvider.Instance.RunQueryAsync(TestServerType.OnPrem, testDbName, dropTableScript);
-                tableChildren = await _service.ExpandNode(session, tablesNode.NodePath);
+                tableChildren = (await _service.ExpandNode(session, tablesNode.NodePath)).Nodes;
                 Assert.True(tableChildren.Any(t => t.Label == "dbo.t1"));
-                tableChildren = await _service.ExpandNode(session, tablesNode.NodePath, true);
+                tableChildren = (await _service.ExpandNode(session, tablesNode.NodePath, true)).Nodes;
                 Assert.False(tableChildren.Any(t => t.Label == "dbo.t1"));
 
             });
         }
 
         [Fact]
+        public async void ExpandOfflineDatabaseShouldReturnError()
+        {
+            var query = "ALTER DATABASE {0} SET OFFLINE WITH ROLLBACK IMMEDIATE";
+            string databaseName = "master";

```

Comment:
will this test leave master offline when if it fails or stops in the middle?  if so we should consider using a different test database.